include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -330,881 +330,1624 @@ struct SPhysicalDeviceFeatures
+    bool robustBufferAccess2 = false;
+    // Nabla Core Profile but still a feature because enabling has overhead
+    bool robustImageAccess2 = false;
+    /*
+    ! nullDescriptor: you can use `nullptr` for writing descriptors to sets and Accesses to null descriptors have well-defined behavior.
+    [TODO] Handle `nullDescriptor` feature in the engine.
+    */
+    bool nullDescriptor = false;
 
-    // [TODO] now we need API to deal with queries and begin/end conditional blocks
-    /* ConditionalRenderingFeaturesEXT *//* VK_EXT_conditional_rendering */
-    bool conditionalRendering = false;
-    bool inheritedConditionalRendering = false;
+    // [DO NOT EXPOSE] not going to expose custom border colors for now
+    /* CustomBorderColorFeaturesEXT *//* VK_EXT_custom_border_color */
 
-    // [EXPOSE AS LIMIT]
-    /* DeviceMemoryReportFeaturesEXT *//* VK_EXT_device_memory_report */
+    // [DO NOT EXPOSE] just reserved numbers, extension never shipped
+    // VK_EXT_extension_289
 
-    // [TODO] need impl
-    /* FragmentDensityMapFeaturesEXT *//* VK_EXT_fragment_density_map */
-    bool fragmentDensityMap = false;
-    bool fragmentDensityMapDynamic = false;
-    bool fragmentDensityMapNonSubsampledImages = false;
-    /* FragmentDensityMap2FeaturesEXT *//* VK_EXT_fragment_density_map2 */
-    bool fragmentDensityMapDeferred = false;
+    // [DO NOT EXPOSE] 0 documentation
+    /* VK_GOOGLE_user_type */
 
-    // [TODO] Investigate later
-    /* Image2DViewOf3DFeaturesEXT *//* VK_EXT_image_2d_view_of_3d */
-    //bool           image2DViewOf3D;
-    //bool           sampler2DViewOf3D;
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_KHR_pipeline_library
+
+    // [DO NOT EXPOSE] just reserved numbers, extension never shipped
+    // VK_NV_extension_292
+
+    // [DO NOT EXPOSE] Triage leftover extensions below    
+    /* VK_NV_present_barrier */
+
+    // [EXPOSE AS A LIMIT] Enabled by Default, Moved to Limits
+    /* VK_KHR_shader_non_semantic_info */
+
+    // [DO NOT EXPOSE] no point exposing until an extension more useful than VK_KHR_present_wait arrives
+    /* PresentIdFeaturesKHR *//* VK_KHR_present_id */
 
     // [DEPRECATED] Vulkan 1.3 core non-optional
-    /* VK_EXT_image_robustness */
+    /* PrivateDataFeatures *//* VK_EXT_private_data */
+
+    // [DO NOT EXPOSE] just reserved numbers, extension never shipped
+    // VK_KHR_extension_297
+
+    // [DO NOT EXPOSE] We don't support yet
+    // VK_EXT_pipeline_creation_cache_control
+
+    // [DO NOT EXPOSE] just reserved numbers, extension never shipped
+    // VK_KHR_extension_299
+
+    // [TODO] Provisional
+    /* VK_KHR_video_encode_queue */
 
-    // [DEPRECATED] Required wholly by ROADMAP 2022 and Nabla Core Profile
-    /* InlineUniformBlockFeaturesEXT *//* VK_EXT_inline_uniform_block */
+    // [DO NOT EXPOSE]
+    /* DiagnosticsConfigFeaturesNV *//* VK_NV_device_diagnostics_config */
+    //bool           diagnosticsConfig;
```

Comment:
remove such commented out members for features we don't expose or expose as limits